### DISCUSSION TITLE
top navigation (to external URLs) for book (and footer too)

### DISCUSSION THREAD

**Discussion by espinielli** [quarto-dev/quarto-cli#713] [Q&A] [answered]
Hi all,
Is there a way to add top navigation to a quarto book?

My goal is to deploy independent books/documents on the web in their own Netlify apps but "integrate" them under the same sister website so that 

https://sister.website.org/books/book1 --> https://book1.netlify.app/
https://sister.website.org/books/book2 --> https://book2.netlify.app/

(the above can be done with "redirect" if the sister.website.org is served as a Netlify app, it is just a matter of configuring it in `netlify.yml`)

where
  https://sister.website.org/
has a fixed top navigation bar (and footer) which I would like to mimic in book1 and book2 top navigation bar.

Any suggestions super welcome.

**Comment by jjallaire** [comment, marked as answer]
Yes, you can add a `navbar` to your `_quarto.yml` file for a book and it will show up on top.

**Comment by espinielli** [comment]
It works like a charm!
Thanks a million.
